src/main/resources/openapi/audiorecording.yaml

```diff
@@ -0,0 +1,130 @@
+openapi: 3.0.1
+info:
+  title: Audio Recording Service
+  version: v0.1.0
+servers:
+  - url: http://localhost:8080
+paths:
+  /audios:
+    post:
+      tags:
+        - audio-recording
```

Comment:
Please change tag to `Audio` such that this generates under `AudioApi.java`.

Reply:
ACCEPT